README.md

```diff
@@ -233,3 +233,9 @@ This library is licensed under the Apache 2.0 license although portions of this
 Specifically, this product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit ([http://www.openssl.org](http://www.openssl.org/))
 and this product also includes cryptographic software written by Eric Young (eay@cryptsoft.com).
 
+TODO (callout for reviewers) -- what to do about licensing differences between
+OpenSSL and AWS-LC? Is there anything meaningful here? It looks like all of
+AWS-LC's antecedents are published under apache 2.0 as well:
+
+https://github.com/awslabs/aws-lc/blob/main/LICENSE#L1-L20
+
```

Comment:
If we are not entirely under Apache 2.0 with no remaining stuff from the old dual OpenSSL/SSLeay license, then lets remove anything in this section not relevant to Apache 2.0. (Or better, note that as of version 2.0, only Apache 2.0 applies.)

Reply:
After reading through AWS-LC's `LICENSE` file more closely, it looks like AWS-LC [still uses](https://github.com/awslabs/aws-lc/blob/main/LICENSE#L8-L20) the combination Apache 2.0 + OpenSSL/SSLeay, as well as adding a few more licensing pieces courtesy of BoringSSL (some for source dependencies, some for test/build dependencies which aren't distributed).

I've added a sentence pointing to this change.